### PR TITLE
[WIP] EZP-27160: Update documentation to match the code

### PR DESCRIPTION
**Pull request by jacek-foremski**
This PR aims to update the documentation to match the code that currently resides in the master branch. 

The fact that there are differences was mentioned by Rui Silva in https://jira.ez.no/browse/EZP-27160. I did the tutorial again step by step and this PR results from the differences I have found.

@mere-teresa @bdunogier please review.

**TODO:**
- [ ] Rebase to the new branches

### CODE REVIEW

File: docs/tutorial/create_the_bundle.md [hunk 1]
```diff
@@ -52,19 +52,24 @@ Target Directory [src/]:<enter>
...
+Controller/
+DependencyInjection/
 EzSystemsTweetFieldTypeBundle.php
-Resources
+Resources/
+Tests/
 ```
 
 Feel free to delete the Controller folder, since you won’t use it in this tutorial. It could have been useful, had our Field Type required an interface of its own.
+Also, you can safely delete Resources/views/Default folder and Resources/config/routing.yml file, as they won't be needed. You should remove "ez_systems_tweet_field_type" entry in your app/config/routing.yml file as well.
```
Comment (DominikaK) [inline docs/tutorial/create_the_bundle.md]: Also, you can safely delete the `Resources/views/Default` folder and `Resources/config/routing.yml`, as they won't be needed. You should remove the `ez_systems_tweet_field_type` entry from your app/config/routing.yml file as well.

File: docs/tutorial/introduce_a_template.md [hunk 2]
```diff
@@ -102,6 +101,30 @@ system:
 
 Notice that the `ezpublish` yaml block was deleted. This is because you already import your configuration under the `ezpublish` namespace in the `prepend` method.
 
+##Adding eztweet_settings block
+
+Currently to be able to display a Content item you need to add twig file with eztweet_settings block. This behaviour is being resolved ([EZP-27344](https://jira.ez.no/browse/EZP-27344)), but for know you have to do two more steps.
```
Comment (DominikaK) [inline docs/tutorial/introduce_a_template.md]: Currently to be able to display a Content item you need to add a twig file with an `eztweet_settings` block. This behavior is being resolved ([EZP-27344](https://jira.ez.no/browse/EZP-27344)), but for now you have to do two more steps.

**Review by DominikaK — COMMENTED** [review]
Just two tiny language fixes, not worth making a separate PR.

**Comment by jacek-foremski** [comment]
@DominikaK thank you! Fixed what you have mentioned.

**Comment by mere-teresa** [comment]
In fact we want to have a separate branch for this PR, right?

**Comment by jacek-foremski** [comment]
@mere-teresa Yes, as soon as it's created I will change the target of this PR.

You can still review it if you want :)

File: docs/tutorial/implement_the_tweet_type_class.md [hunk 3]
```diff
@@ -190,43 +189,44 @@ public function validateValidatorConfiguration( $validatorConfiguration )
...
+    $errors = [];
+
+    if ($this->isEmptyValue($fieldValue)) {
+        return $errors;
+    }
+
+    // Tweet Url validation
+    if (!preg_match('#^https?://twitter.com/([^/]+)/status/[0-9]+$#', $fieldValue->url, $m)) {
+        $errors[] = new ValidationError(
+            'Invalid twitter status url %url%',
```
Comment (mere-teresa) [inline docs/tutorial/implement_the_tweet_type_class.md]: Nitpick: Twitter (first UC), and URL (acronym)

File: docs/tutorial/implement_the_tweet_value_class.md [hunk 4]
```diff
@@ -26,23 +26,24 @@ You will end up with the following properties:
...
+ * @var string
+ */
 public $url;
 
-
 /**
-* Author's tweet URL (http://twitter.com/UserName)
-* @var string
-*/
+ * Author's tweet URL (http://twitter.com/UserName)
```
Comment (mere-teresa) [inline docs/tutorial/implement_the_tweet_value_class.md]: Isn't this URL https:// ?

**Review by mere-teresa — APPROVED** [review]
Thank you for this work. Great job!

File: docs/tutorial/implement_the_tweet_value_class.md (hunk 4 above)
Comment (jacek-foremski) [reply docs/tutorial/implement_the_tweet_value_class.md]: It can be both, but the http redirects to https. Will fix.

**Comment by bdunogier** [comment]
What is the status of this pull-request, @natanael89 ? It looks like it is ready for merge. Given the commits list, you don't need to squash it, we can do it from the github UI.

**Comment by jacek-foremski** [comment]
I would like to wait until we are done with the documentation branch, so it can be merged there. It's getting hard to track changes to all branches :)

**Comment by timiTao** [comment]
@natanael89 any progress with that?

**Comment by jacek-foremski** [comment]
You can track the progress here: #19 and here: [EZP-27292](https://jira.ez.no/browse/EZP-27292). As soon as they're finished, I will rebase this PR onto new branches.

**Pull request closed without merging**